media/css/sandstone/buttons.less

```diff
@@ -345,267 +343,6 @@ html[lang^='en'] {
     }
 }
 
-.download-button .download-link {
```

Comment:
Note to self: various parts of `buttons.less` are being removed here, but the file itself no longer seems to be used in sandstone with this change. Do we still need any of it?